routers/api/user/createUser.js

```diff
@@ -1,16 +1,18 @@
-const models=require('../../../models')
-
+const models = require('../../../models')
+const Logger = require('../../../services/logger')
+const logger = new Logger('signup')
 async function createUser(req, res, next) {
     try {
         const user = await models.User.create(req.body)
-
+        logger.info("success")
```

Comment:
Include timestamp, and requested rouute in logger in info and error

Reply:
timestamp and route is included in logger.js of services folder